lib/WebpackOptionsApply.js

```diff
@@ -450,7 +462,11 @@ class WebpackOptionsApply extends OptionsApply {
 		}
 		if (options.optimization.minimize) {
 			for (const minimizer of options.optimization.minimizer) {
-				minimizer.apply(compiler);
+				if (typeof minimizer === "function") {
+					minimizer.apply(compiler);
+				} else {
+					minimizer.apply(compiler);
+				}
```

Comment:
@sokra is this a typo or am I missing something?  There's a similar bit further down with `plugin.apply(compiler)`